pkg/apis/apis/v1alpha1/types.go

```diff
@@ -193,6 +193,10 @@ const (
 	// for the request. This data is synthetic; it is not stored in etcd and instead is only applied when retrieving
 	// CRs for the CRD.
 	AnnotationAPIIdentityKey = "apis.kcp.dev/identity"
+
+	// AnnotationDefaultCreated is the annotation key for an apiexport or apibinding indicating the other default resources
+	// has been created already. If the created default resource is deleted, it will not be recreated.
+	AnnotationDefaultCreated = "apis.kcp.dev/default-created"
```

Comment:
bike-shed: `apis.kcp.dev/skip-default-object-creation`. Thinking that maybe user will want to use this in Gitops context.